components/states/ReservationTypeState.jsx

```diff
@@ -55,6 +58,7 @@ const ReservationTypeState = () => {
           }
         />
       </StateCards>
+      <LanguageSelector />
```

Comment:
we should put this on the page, not on a state I think (that way they can toggle it from the page level)